### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found few misspellings.

@JoshData https://github.com/crate-ci/typos

### CODE REVIEW

File: setup/migrate.py [hunk 1]
```diff
@@ -23,11 +23,11 @@ def move_file(fn, domain_name_escaped, filename):
 	# Migrate the 'domains' directory.
 	for sslfn in glob.glob(os.path.join( env["STORAGE_ROOT"], 'ssl/domains/*' )):
 		fn = os.path.basename(sslfn)
-		m = re.match("(.*)_(certifiate.pem|cert_sign_req.csr|private_key.pem)$", fn)
+		m = re.match("(.*)_(certificate.pem|cert_sign_req.csr|private_key.pem)$", fn)
```
Comment (szepeviktor) [inline setup/migrate.py:26]: ⚠️ This needs attention.

Comment (JoshData) [reply setup/migrate.py:26]: Yep, that should be reverted. The typo is documented in the code comments around there.

Comment (szepeviktor) [reply setup/migrate.py]: Done!

**Comment by JoshData** [comment]
TY

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**